src/modules/core/components/CountDownTimer/calculateTimeLeft.ts

```diff
@@ -0,0 +1,18 @@
+export const calculateTimeLeft = (createdAt: number, actionPeriod?: number) => {
```

Comment:
This method should go into `utils/time`

_(you'll need to create the file + folder structure for this)_